Add boilerplate docs to allow inclusion into docs.scs. (#3)
SovereignCloudStack/security-infra-scan-pipeline: docs/overview.md

# SCS automated pentesting

Security has a high priority in modern cloud infrastructures. If you look at it from an automation perspective, a basic distinction of tooling is needed to be considered:

* SAST or Static Application Security Testing: These tools scan code that is checked into e.g. git and are integrated into build pipelines. They only look at static artifacts: code, dependencies, container images.

> * DAST or Dynamic Application Security Testing: With dynamic testing, running programs and deployed infrastructure is scanned. This allows to test infrastructure configuration and seeing behaviour of running processes.

Reviewer: ```suggestion
* DAST or Dynamic Application Security Testing: With dynamic testing, running programs and deployed infrastructure are scanned. This allows to identify vulnerabilities, test infrastructure configuration and analyze the behaviour of running processes.
```
